lib/getValidUrlFromInput.ts

```diff
@@ -1,7 +1,7 @@
 export function getValidUrlFromInput(urlString: string): URL {
-  if (urlString.includes('#')) {
+  if (/#\d+/.test(urlString)) {
     urlString = urlString.replace('#', '/issues/')
-  } else if (!urlString.includes('issues')) {
+  } else if (!/\/issues\/\d+/.test(urlString)) {
```

Comment:
i don't think we want case insensitive, because https://github.com/pln-planning-tools/Starmap/Issues/237 is an invalid link, while https://github.com/pln-planning-tools/Starmap/issues/237 is not.